lms/djangoapps/teams/tests/test_views.py

```diff
@@ -129,7 +129,7 @@ def test_query_counts(self):
         team.add_user(self.user)
 
         # Check the query count on the dashboard again
-        with self.assertNumQueries(19):
+        with self.assertNumQueries(20):
```

Comment:
The number of queries should not go up.  The view/serializer needs to use `select_related` or `prefetch_related` so the additional data is retrieved with a single database query.